example/src/App.js

```diff
@@ -91,3 +116,14 @@ function NotFound({ params: { '*': path } }) {
     </div>
   )
 }
+
+function usePageRoutePath() {
+  const pick = usePick()
+  const { params, route } = pick(
```

Comment:
Zouden we hier nog iets voor kunnen verzinnen waar bij je niet zelf nog al je routes hier in moet zetten? Als dat zou kunnen zou je er misschien een hook van kunnen maken die in deze library zit?

Dit zou volgens mij moeten werken?

```js
const { params, route } = pick(...Object.values(routeMap))
```

Reply:
Nee, het lijstje met routes dat je mee wilt laten doen verschilt per routemap. In dit geval zitten `articles` en `articles.article` bijvoorbeeld niet op hetzelfde niveau.